core/primitives/src/shard_layout.rs

```diff
@@ -1303,391 +1152,6 @@ mod tests {
         ShardLayout::v2(boundary_accounts, shard_ids, shards_split_map)
     }
 
-    #[test]
-    fn test_shard_layout_all() {
-        #[allow(deprecated)]
-        let v0 = ShardLayout::v0(1, 0);
-        let v1 = ShardLayout::get_simple_nightshade_layout();
-        let v2 = ShardLayout::get_simple_nightshade_layout_v2();
-        let v3 = ShardLayout::get_simple_nightshade_layout_v3();
-        let v4 = ShardLayout::get_simple_nightshade_layout_v4();
-        let v5 = ShardLayout::get_simple_nightshade_layout_v5();
-        let v6 = ShardLayout::get_simple_nightshade_layout_v6();
-
-        insta::assert_snapshot!(serde_json::to_string_pretty(&v0).unwrap(), @r###"
```

Comment:
nice, can we now remove serde from the ShardLayout structs?

Reply:
Will follow up offline